pudl/datastore.py

```diff
@@ -191,40 +244,142 @@ def download(source, year, datadir=settings.DATA_DIR, verbose=True):
 
     Args:
         source (str): the data source to retrieve. Must be one of: 'eia860',
-            'eia923', or 'ferc1'.
+            'eia923', 'ferc1', or 'epacems'.
         year (int): the year of data that the returned path should pertain to.
             Must be within the range of valid data years, which is specified
-            for each data source in pudl.constants.data_years.
+            for each data source in pudl.constants.data_years. Note that for
+            data (like EPA CEMS) that have multiple datasets per year, this
+            function will download all the files for the specified year.
```

Comment:
Seems like for testing (and general sanity) purposes we may want to allow only a subset of states to be downloaded and loaded for CEMS.